Src/Eiffel/interface/new_graphical/text_window/text/autocomplete/eb_class_info_analyzer.e

```diff
@@ -435,6 +435,83 @@ feature {NONE}-- Clickable/Editable implementation
 			end
 		end
 
+	described_local (token: EDITOR_TOKEN; line: EDITOR_LINE; ft: FEATURE_AS; ft_start_token: detachable EDITOR_TOKEN_FEATURE_START): detachable TUPLE [feat: E_FEATURE; ast: detachable AST_EIFFEL]
+			-- search in feature represented by `ft' the feature associated with `token' if any
+		require
+			token_not_void: token /= Void
+			line_not_void: line /= Void
+			token_in_line: line.has_token (token)
+		local
+			l_type: TYPE_A
+			vn: READABLE_STRING_GENERAL
+			vn_id: INTEGER
+			td: detachable AST_EIFFEL
+		do
+			if is_ok_for_completion then
+				initialize_context
+				if attached current_class_c as cl then
+					if not token_image_is_in_array (token, unwanted_symbols) then
+						if ft /= Void then
+							current_feature_as := [ft, ft.feature_names.first]
+						else
+							current_feature_as := Void
+						end
+						error := False
+						last_was_constrained := False
+						last_feature := Void
+						if attached cl.feature_with_name_id (ft.feature_name.name_id) as feat then
```

Comment:
Does pick and drop on local also works in the flat view of a renamed routine if you are just using the name? Also don't we get a call on Void-target if `ft is Void`.